cmd/query/app/grpc_handler_test.go

```diff
@@ -511,10 +512,13 @@ func TestGetDependenciesSuccessGRPC(t *testing.T) {
 	withServerAndClient(t, func(server *grpcServer, client *grpcClient) {
 		expectedDependencies := []model.DependencyLink{{Parent: "killer", Child: "queen", CallCount: 12}}
 		endTs := time.Now().UTC()
+		expectedEndTs := endTs.Add(time.Duration(-1) * defaultDependencyLookbackDuration)
```

Comment:
why is expectedEndTs offset from endTs?

Reply:
should it be called startTs perhaps?

Reply:
also, does `Add(-defaultDependencyLookbackDuration)` work?

Reply:
@yurishkuro Its a bit weird because here startTime is passed into endTs (https://github.com/jaegertracing/jaeger/blob/main/cmd/query/app/grpc_handler.go#L226)

Reply:
it looks like a bug

Reply:
opened a patch at https://github.com/jaegertracing/jaeger/pull/6324

Reply:
@yurishkuro should be good to go now